ci/jenkins/jobs/job-templates.yaml

```diff
@@ -425,3 +425,28 @@
         started-status: '{started_status}'
     wrappers: '{wrappers}'
 
+- job-template:
+     name: 'cloud-{name}-{test_name}-aws'
```

Comment:
We don't need a new job template, just change the test_name in project yaml.

Reply:
The same reason: https://github.com/antrea-io/antrea/pull/6922#discussion_r1959097318